libs/core/src/components/alert/alert.stories.ts

```diff
@@ -0,0 +1,210 @@
+import { Meta, StoryObj } from '@storybook/web-components'
+
+import { html } from '../../utils/helpers/custom-element-scoping'
+import { GdsAlert } from './alert.component'
+
+import './index.ts'
+import '../card/card.ts'
+import '../button/button.ts'
+
+const meta: Meta<GdsAlert> = {
+  title: 'Components/Alert',
+  component: 'gds-alert',
+  tags: ['autodocs'],
+  argTypes: {
+    type: {
+      control: 'select',
+      options: ['info', 'success', 'warning', 'error'],
+    },
+    showIcon: { control: 'boolean' },
+    dismissible: { control: 'boolean' },
+    timeOut: { control: 'number' },
+    buttonText: { control: 'text', name: 'Button Text (CTA)' },
+  },
+}
+
+export default meta
+
+type Story = StoryObj<GdsAlert>
+
+const Template = (args: any, { parameters }: any) => html`
+  <gds-div padding="xs; l {m}" margin="m">
```

Comment:
I think the story should show exactly how the component renders, so remove this div to show the "reality"